### PR TITLE
UI file details

### PR DESCRIPTION
**Pull request by PaddleStroke**

### CODE REVIEW

File: Utils.py [hunk 1]
```diff
@@ -6,6 +6,9 @@
 
 import os
 import FreeCAD
+import zipfile
+from PySide2.QtGui import QPixmap
+modPath = os.path.dirname(__file__).replace("\\", "/")
```
Comment (amrit3701) [inline Utils.py:11]: Same comment: https://github.com/Ondsel-Development/Ondsel-Lens/pull/36#pullrequestreview-1481932139

File: Utils.py [hunk 2]
```diff
@@ -26,4 +29,25 @@ def isOpenableByFreeCAD(filename):
             ext = ext[1:]
     if ext in extensions:
         return True
-    return False
\ No newline at end of file
+    return False
```
Comment (amrit3701) [inline Utils.py]:
```python
    if ext in extensions:
        return True
    return False
```

One liner:

```python
    return ext in extensions
```

**Review by amrit3701 — CHANGES_REQUESTED** [review]

File: Utils.py (hunk 1 above)
Comment (PaddleStroke) [reply Utils.py:11]: This is a widespread topic it appears. So maybe it can be changed in all files in a later PR to avoid conflicts.

**Comment by sliptonic** [comment]
The PR needs to be rebased from the current master.

**Comment by PaddleStroke** [comment]
Rebased on current master

**Pull request merged**